build/configs/rtl8721csm/rtl8721csm_download.sh

```diff
@@ -18,366 +18,49 @@
 ###########################################################################
 # rtl8721csm_download.sh
```

Comment:
Could you please amend the commit title as: -

> build/configs/rtl8721csm: Add common download script

Similar to the title for the commit on imxrt changes?

Reply:
Done.